serverless/functions/quickstart.mdx

```diff
@@ -46,17 +46,21 @@ The Scaleway [Elements Functions platform](https://www.scaleway.com/en/serverles
 2. Click the functions namespace you want to use to create your function.
 3. Click **Create function**. The function creation wizard displays.
-4. Complete the following steps in the wizard:
-    - **Setup function**:
-      - From the first drop-down menu, choose to either use the inline code editor or upload a `*.zip` archive [(zip requirements)](/serverless/functions/how-to/package-function-dependencies-in-zip/).
-      - From the second drop-down menu, choose your [desired runtime](/serverless/functions/reference-content/functions-lifecycle/).
-      - From the third drop-down menu, set the Handler for your function
+4. Complete the following steps in the wizard: 
+    - Enter your **function code**. 
+      - Set the handler for your function.
+      - Choose your desired runtime and its version.
+      - Choose to either use the inline code editor to edit your code directly from your browser in the box provided, or to upload a `*.zip` archive.
     - Choose a **name** for your function, and optionally a **description**. The name must only contain alphanumeric characters.
     - Choose the **resources** to be allocated to your function at runtime. These define the performance characteristics of your function.
     - Set your **scaling** preferences, or leave them at default values. The Scaleway platform autoscales the number of available instances of your function to match the incoming load, depending on the settings you define here.
-    - Define **environment variables** you want to inject into your function. For each environment variable, click **+Add** and enter the key/value pair.
+    - Define any **environment variables** you want to inject into your function. For each environment variable, click **+Add** and enter the key/value pair.
+    - Set secret environment variables (optional). **Secrets** are environment variables which are injected into your function and stored securely, but not displayed in the console after initial validation. Add a **key** and a **value**.
     - Set the desired **privacy policy** for your function. This defines whether a function can be executed anonymously (**public**) or only via an authentication mechanism provided by the [Scaleway API](https://www.scaleway.com/en/developers/api/serverless-functions/#authentication) (**private**).
-    - Optionally, use the **advanced options** to add [triggers](/serverless/functions/concepts/#trigger) to your function deployment. Adding triggers enables the function to handle events coming from sources other than an HTTP endpoint, such as CRON (time-based jobs).
+    - Click **Advanced Options** if you wish to set a custom timeout for your function.
+      <Message type="note">
+        [Triggers](/serverless/functions/concepts/#trigger) can no longer be added from this screen. Refer to [this page](/serverless/functions/how-to/add-trigger-to-a-function/) to add a trigger to your function.
```

Comment:
```suggestion
        [Triggers](/serverless/functions/concepts/#trigger) can no longer be added from this screen. For more information, refer to the [How to add a trigger to a function](/serverless/functions/how-to/add-trigger-to-a-function/) documentation.
```
For accessibility reasons we try to include the page title in the hyperlink as much as we can 🤓